rudder-lang/src/bin/rudderc.rs

```diff
@@ -88,7 +88,7 @@ fn main() {
     };
     let command_status = command_result.is_ok();
```

Comment:
naming it `command_success` or something like that would have prevented the mistake.

Reply:
True, fixuped. And added an explicit error for cli errors